init

```diff
@@ -50,15 +62,26 @@ modprobe dm-mod
 vgscan --mknodes --ignorelockingfailure >/dev/null 2>&1 || rescue_shell
 vgchange --sysinit --activate y >/dev/null 2>&1 || rescue_shell
 
+msg "Scanning for live media detection..."
+sfs="seiba/seiba.sfs"
+while [ "$live" = "1" ] && [ "$root" == "" ] ; do
+	list=$(ls /sys/class/block/ | grep ".*[0-9]$" | grep -v loop | grep -v ram | grep -v nbd | sed "s|^|/dev/|g")
+	for part in $list
+	do
+		sleep 0.1
+		if is_file_avaiable "$part" "${sfs}"
+		then
+			msg "Detected live media $part"
+			export root=$part
+		fi
+	done
+done
+
+
```

Comment:
list all avaiable parts and scan file in partition. 
sleep 0.1 provide mount delay

Reply:
will check